sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/TablesLiveTestBase.cs

```diff
@@ -100,7 +100,7 @@ protected async Task<T> CallAsync<T>(string methodName = null, object arguments
         protected async Task<object> CallAsync(Type funcType, string methodName = null, object arguments = null, Action<HostBuilder> configure = null)
         {
             var instance = Activator.CreateInstance(funcType);
-            var (host, jobHost) = CreateHost(funcType, configure, instance);
+            using var jobHost = CreateHost(funcType, configure, instance);
```

Comment:
I eventually reproed this on WSL and was able to collect a crash dump. I saw there was an OOM and StackOverflow exception. I didn't pinpoint the exact code, but using that clue I looked more closely at the code and realized we were not disposing the job host.

Reply:
Do we want to keep the dump changes, or just the fix? If not, perhaps we could put these dump changes behind a variable config so that it's easy to turn on as needed.

Reply:
I think it is worth keeping the dump collection on. Do you have any concerns with it?
/cc @weshaggard

Reply:
I like keeping the dump code as well.